src/database/DataTypes/TemperatureBreachConfiguration.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import Realm from 'realm';
+
+export class TemperatureBreachConfiguration extends Realm.Object {}
+
+TemperatureBreachConfiguration.schema = {
+  name: 'TemperatureBreachConfiguration',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    minimumTemperature: { type: 'double', optional: true },
+    maximumTemperature: { type: 'double', optional: true },
+    duration: { type: 'double', optional: true },
+    description: { type: 'string', optional: true },
+    colour: { type: 'string', optional: true },
+  },
+};
```

Comment:
Same with this - could possibly just not be a class but it might be nice to have a few methods for checking if something is a breach in here? 

Also: I imagine `duration` being miliseconds - `minTemperature` or `minimumTemperature`?

Reply:
🤔 `durationInMilliseconds` ( just being annoying. duration is fine as it is. )
or `minimumTemp`!

I would generally use min & max myself but perversely prefer the `minimum`. 
More descriptive and less ambiguous